styles/main.css

```diff
@@ -0,0 +1,968 @@
+    margin-top: 0;
+    font-weight: 500;
+    margin-bottom: 2rem;
+}
+
+@media(min-width: 550px) {
+    .team {
+        padding: 3rem 3rem;
+    }
+    .team__member {
+        padding: 0 1rem;
+    }
+    .team__list {
+        grid-template-columns: auto auto;
+        grid-template-rows: auto;
+        max-width: 90%;
+    }
+}
+
+@media(min-width: 800px) {
+    .team {
+        padding: 5rem 8rem;
+    }
+    .team__list {
+        grid-template-columns: auto auto auto;
+        max-width: 80%;
+    }
+}
+
+
+/* Portfolio */
+
+.portfolio {
+    text-align: center;
+    padding: 2rem 2rem;
+    background: var(--gray-background);
+}
+
+.portfolio__list {
+    padding: 0;
+    list-style: none;
+    display: grid;
+    grid-template-rows: repeat(6, auto);
+    grid-template-columns: auto;
+    grid-gap: 2rem;
+}
+
+.portfolio__item {
+    position: relative;
+    display: flex;
+    flex-direction: column-reverse;
+    height: 100%;
+}
+
+.portfolio__item img {
+    width: 100%;
+    height: 100%;
+}
+
+.portfolio__hover {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background: var(--light-green);
+    opacity: 0;
+    transition: opacity 1s ease-out;
+}
+
+.portfolio__hover-icon {
+    border-radius: 50%;
+    height: 4rem;
+    width: 4rem;
+    padding-left: 1rem;
+    text-align: center;
+    border: white;
+    background: white;
+    color: var(--light-green);
+}
+
+.portfolio__hover h1,
+.portfolio__hover h2 {
+    color: white;
+}
+
+@media(min-width: 550px) {
+    .portfolio {
+        padding: 3rem 3rem;
+    }
+    .portfolio__list {
+        grid-template-columns: auto auto;
+        grid-template-rows: repeat(3, auto);
+    }
+}
+
+@media(min-width: 800px) {
+    .portfolio {
+        padding: 5rem 8rem;
+    }
+    .portfolio__list {
+        grid-template-columns: auto auto auto;
+        grid-template-rows: repeat(2, auto);
+    }
+}
+
+
+/* Our Blog*/
+
+.blog {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem 2rem;
+}
+
+.blog__list {
+    width: 100%;
+    text-align: left;
+    list-style: none;
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: repeat(4, auto);
+    grid-gap: 2rem;
+    padding: 0;
+}
+
+.blog__item {
+    padding: 3rem 2rem;
+    border: solid lightgray 1px;
+    border-radius: 0.5rem;
+    background: white;
+}
+
+.blog__item h3 {
+    color: darkgray;
+    font-size: 1rem;
+}
+
+.blog__item h1 {
+    color: gray;
+    font-size: 1.4rem;
+}
+
+.blog__item h2 {
+    color: gray;
+    font-size: 1rem;
+}
+
+.blog__item:hover,
+.blog__item:active {
+    background: url("../images/hover-image.jpg") top left/cover no-repeat;
+    animation: blog 15s linear infinite;
+}
+
+.blog__item:hover h1,
+.blog__item:active h1 {
+    color: var(--light-green);
+}
+
+@keyframes blog {
```

Comment:
describe the animation instead of the section it is used on, as it might be reused